bootloader/src/nRF52840/usbd_device.cpp

```diff
@@ -143,11 +143,11 @@ class NrfDevice : public Device {
 #elif PLATFORM_ID == 14
 # define USBD_PRODUCT_NAME              "Xenon"
 #elif PLATFORM_ID == 22
-# define USBD_PRODUCT_NAME              "Argon SoM"
+# define USBD_PRODUCT_NAME              "A SoM"
 #elif PLATFORM_ID == 23
-# define USBD_PRODUCT_NAME              "Boron SoM"
+# define USBD_PRODUCT_NAME              "B SoM"
 #else
```

Comment:
I know this isn't the change you just made, but shouldn't this be narrowed to `PLATFORM_ID == 24`?

Reply:
I see what you mean, and I'm not sure what we would default it to then... at least this is a somewhat sane default as there are only 6 nrF52840 platforms.

Reply:
I guess we could just throw an #error "platform not supported"